FSEParser_V3.1.py

```diff
@@ -74,216 +72,229 @@
     0x00800000: 'NOT_USED-0x00800000;'
 }
 
-print '\n=========================================================================='
-print 'FSEParser v', VERSION, ' -- provided by G-C Partners, LLC'
-print '==========================================================================\n'
+print('\n==========================================================================' '\n'

```

Comment:
Adding the parenthesis to the print statements result in the parenthesis being printed along with the str. Instead of using the parenthesis to try and conform to Python 3 standards, they may need to be removed. Even with the parenthesis in the print statements, running the script using python 3 still fails. This applies to all print states in the script. Refer to comment on line 128/129 for suggestion on testing if python 3 is in use.